pokesets/Manga/Pokemon Adventures/PS-BattleFactory.yaml

```diff
@@ -0,0 +1,84 @@
+species: Linoone
+setname: PS-Battle Factory
+ingamename: Linoone
+displayname: Linoone
+ability: [Pickup]
+item: [Choice Band]
+ball: [Poke]
+moves:
+- [Frustration]
+- [Trick]
+- [Dig]
+- [Thunder Wave]
```

Comment:
I know it's a consequence of using canonical moves, but Choice Band on Thunder Wave is ehhhhh. I'd suggest turning down the PP for this one so that if it occurs during defiance, there's less chance of a lockout.